src/server.rs

```diff
@@ -360,6 +363,39 @@ impl Server {
         self.shutdown_handler = Some(Arc::new(function));
         debug!("Added shutdown handler {:?}", self.shutdown_handler);
     }
+
+    /// Get and set call list from memory store
+    pub fn get_calls_list(
+        self_: &PyCell<Self>,
+        limit_key: String,
+        limit_ttl: u32,
+        current_timestamp: u32,
+    ) -> Vec<u32> {
```

Comment:
Can this just not return the length? We're doing expensive clones here, and only ever using the length python side